lib/status-wd.js

```diff
@@ -21,10 +22,11 @@ module.exports = function(parentCtl) {
 
   if (!parentCtl) return;
 
-  cluster.on('fork', function(worker) {
+  master.on('fork', function(worker) {
     worker.on('message', function(msg) {
       if (msg.cmd !== 'status:wd') return;
       debug('master received: %j', msg);
+      msg.pst = worker.startTime;
```

Comment:
`.pst` property doesn't need to be part of the status:wd message, does it? Or are you trying to make sure `pid/wid/pst` triplet is every message that contains worker-identifying information?